### PR TITLE
docs: updated with example on how to setup lazymc with minecraft container

### PR DESCRIPTION
**Pull request by joesturge**
I have created a docker container which wraps LazyMc add adds the functionality to control a minecraft container

I have updated the docs to add this as an example. Feel free to check out the project here: https://github.com/joesturge/lazymc-docker-proxy

image

### CODE REVIEW

File: docs/misc/examples.md [hunk 1]
```diff
@@ -26,6 +26,54 @@ services:
 
 [Source](https://github.com/itzg/docker-minecraft-server/blob/master/examples/geyser/docker-compose.yml)
 
+## Lazymc - Put your Minecraft server to rest when idle
+
+With [lazymc-docker-proxy](https://github.com/joesturge/lazymc-docker-proxy/settings) you are able to use [lazymc](https://github.com/timvisee/lazymc) with the minecraft container.
```
Comment (itzg) [inline docs/misc/examples.md:31]: I think meant the link to be:

```suggestion
With [lazymc-docker-proxy](https://github.com/joesturge/lazymc-docker-proxy) you are able to use [lazymc](https://github.com/timvisee/lazymc) with the minecraft container.
```

File: docs/misc/examples.md [hunk 2]
```diff
@@ -26,6 +26,54 @@ services:
...
+```yaml
+services:
+    lazymc:
+        container_name: lazymc
+        image: ghcr.io/joesturge/lazymc-docker-proxy:latest
+        environment:
+            # Point to the service name of the Minecraft server
+            - SERVER_ADDRESS=mc:25565
+            # Required to find the container to manage it
+            - LAZYMC_GROUP=mc
```
Comment (itzg) [inline docs/misc/examples.md:42]: For consistency can you switch this to object syntax:

```suggestion
            # Point to the service name of the Minecraft server
            SERVER_ADDRESS: mc:25565
            # Required to find the container to manage it
            LAZYMC_GROUP: mc
```

**Review by itzg — COMMENTED** [review]
Thanks! Just a couple of small changes.

File: docs/misc/examples.md (hunk 1 above)
Comment (joesturge) [reply docs/misc/examples.md]: I have updated the link, thanks!

File: docs/misc/examples.md (hunk 2 above)
Comment (joesturge) [reply docs/misc/examples.md]: fixed inconsistancies

**Review by itzg — APPROVED** [review]
Thanks for the updates

**Comment by joesturge** [comment]
Cheers! Happy to help

**Comment by eyes-eyes** [comment]
Hello itzg directed me to come here for support.
I am using a ubuntu server and trying to use lazymc proxy. I coped the example configuration from the readme and added my mc environment variables on top of it. Only the lazymc-proxy container is running but still can't connect to the server. 
When I log the lazymc-proxy container it keeps saying this:

> ERROR mc::lazymc::monitor             > Force killing server, took too long to start or stop
> ERROR lazymc-docker-proxy::health     > Application is unhealthy.
> WARN  mc::lazymc                      > Failed to force kill server

**Comment by joesturge** [comment]
> Hello itzg directed me to come here for support. I am using a ubuntu server and trying to use lazymc proxy. I coped the example configuration from the readme and added my mc environment variables on top of it. Only the lazymc-proxy container is running but still can't connect to the server. When I log the lazymc-proxy container it keeps saying this:
> 
> > ERROR mc::lazymc::monitor             > Force killing server, took too long to start or stop
> > ERROR lazymc-docker-proxy::health     > Application is unhealthy.
> > WARN  mc::lazymc                      > Failed to force kill server
> 
> When I try to join from minecraft just keeps saying "Server is starting ... try to reconnect in a minute"

Hi @eyes-eyes I think someone else has expeienced this could you try my suggestion here: https://github.com/joesturge/lazymc-docker-proxy/issues/115#issuecomment-2448597328

**Comment by joesturge** [comment]
> > Hello itzg directed me to come here for support. I am using a ubuntu server and trying to use lazymc proxy. I coped the example configuration from the readme and added my mc environment variables on top of it. Only the lazymc-proxy container is running but still can't connect to the server. When I log the lazymc-proxy container it keeps saying this:
> > > ERROR mc::lazymc::monitor             > Force killing server, took too long to start or stop
> > > ERROR lazymc-docker-proxy::health     > Application is unhealthy.
> > > WARN  mc::lazymc                      > Failed to force kill server
> > 
> > 
> > When I try to join from minecraft just keeps saying "Server is starting ... try to reconnect in a minute"
> 
> Hi @eyes-eyes I think someone else has expeienced this could you try my suggestion here: [joesturge/lazymc-docker-proxy#115 (comment)](https://github.com/joesturge/lazymc-docker-proxy/issues/115#issuecomment-2448597328)

Oh just realised that is infact also you... lol

**Pull request merged**